dGame/dComponents/DestroyableComponent.cpp

```diff
@@ -137,26 +137,15 @@ void DestroyableComponent::Serialize(RakNet::BitStream* outBitStream, bool bIsIn
 
             if (m_IsSmashable) {
                 outBitStream->Write(m_HasBricks);
-
-                if (m_ExplodeFactor != 1.0f) {
-                    outBitStream->Write1();
-                    outBitStream->Write(m_ExplodeFactor);
-                } else {
-                    outBitStream->Write0();
-                }
+				outBitStream->Write(m_ExplodeFactor != 1.0f);
+				if (m_ExplodeFactor != 1.0f) outBitStream->Write(m_ExplodeFactor);
             }
         }
-
 		m_DirtyHealth = false;
     }
 
-    if (m_DirtyThreatList || bIsInitialUpdate) {
-        outBitStream->Write1();
-        outBitStream->Write(m_HasThreats);
-		m_DirtyThreatList = false;
```

Comment:
do we need to set this back to false?

Reply:
Does it _need_ to?  no.  This value is always false and never changes in the source code atm.  For brevity I will set it to false anyways.